pages/api/getUser.ts

```diff
@@ -18,13 +18,7 @@ interface APIOptions {
   data: Object;
 }
 
-interface APIResult {
-  success: boolean;
-  token?: string;
-  error?: string;
-}
-
-async function getResult({ path, data }: APIOptions): Promise<APIResult> {
+async function getResult({ path, data }: APIOptions): Promise<string[]> {
   try {
```

Comment:
Is this errorHandler function to prevent try catch hell? If it is, you should remove try catch cause await have its own catch